docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -77,7 +77,7 @@ This permission allows giving permissions to new addresses. This role-management
         <b>value = </b><code>0x0000000000000000000000000000000000000000000000000000000000000004</code>
     </p>
 
-This permission allows for **editing permissions** of any address that already has some permissions set on the ERC725Account (including itself).
+This permission allows for **editing permissions** of any address that already has some permissions set on the ERC725Account (including itself). One would also need this permission to **change or remove** an existing _controller address_ from `AddressPermission[index]`.
```

Comment:
Same recommendation as above. Here are my suggested changes.

```suggestion
This permission allows for **editing permissions** of any address that has some permissions already set on the ERC725Account (including itself). 
The `CHANGEPERMISSIONS` is also needed to:
- 🗑️ **Remove** a controller `address` from the list of `AddressPermissions[]`, meaning:
    - removing its address at a specific index at `AddressPermissions[index]`
    - decreasing the `AddressPermissions[]` Array length (to describe that a controller address has been removed).
- 🖊️ **Edit** an entry in the `AddressPermissions[index]` Array, meaning changing the address stored at a specific index.

> ⚠️ **Warning:** a controller address with `CHANGEPERMISSIONS` can also edit its own permissions. Be cautious when granting this permission!
```

@YamenMerhi @skimaharvey let me know your feedbacks about the **Warning note** for "editing permissions to self". Might be important to document this.

Reply:
It looks good to me and the mention is important imo